docs/contributing/release-process.md

```diff
@@ -52,13 +52,13 @@ In order to create the first beta release for a new minor version and a new rele
 
 ### Send Pull Request to propose a release
 
-See [1.22.0-beta.2 PR](https://github.com/kubernetes/kops/pull/12467) for an example.
+See [1.25.3-beta.2 PR](https://github.com/kubernetes/kops/pull/12467) for an example.
```

Comment:
I'm not sure updating this doc in the release commit makes sense, but if so the PR should change as well.

Reply:
Yes, was just notes. Removed.